dashed/assets/javascripts/welcome.js

```diff
@@ -0,0 +1,53 @@
+var $ = window.$ = require('jquery');
+var jQuery = window.jQuery = $;
+var px = require('./modules/dashed.js');
+
+require('../stylesheets/dashed.css');
+require('../stylesheets/welcome.css');
+
+require('bootstrap');
+require('datatables');
+require('d3');
```

Comment:
does this export a global `d3` var for the calendar? i.e., does it not need to be `var d3 = require('d3');`?